bitmovin/encoding/statistics.ts

```diff
@@ -83,9 +83,34 @@ export const statistics = (configuration, httpClient: HttpClient) => {
           const url = urljoin(configuration.apiBaseUrl, 'encoding/statistics/encodings', encodingId);
           return get(configuration, url);
         },
-        liveStatistics: () => {
-          const url = urljoin(configuration.apiBaseUrl, 'encoding/statistics/encodings', encodingId, 'live-statistics');
-          return get(configuration, url);
+        liveStatistics: {
+          list: () => {
```

Comment:
this is a breaking change, this should work: `.liveStatistics()`, `.liveStatistics.events.list()`, `.liveStatistics. streams.list()`